src/Compilers/VisualBasic/Portable/Symbols/AnonymousTypes/SynthesizedSymbols/AnonymousTypeOrDelegateTypeParameterSymbol.vb

```diff
@@ -108,6 +108,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Friend Overrides Sub EnsureAllConstraintsAreResolved()
             End Sub
 
+            Public Overrides Function GetHashCode() As Integer
+                Return System.Runtime.CompilerServices.RuntimeHelpers.GetHashCode(Me)
+            End Function
+
+            Public Overrides Function Equals(other As TypeSymbol, comparison As TypeCompareKind) As Boolean
+                Return other Is Me
```

Comment:
Just to confirm, we don't need to compare _container and _ordinal?

Reply:
> Just to confirm, we don't need to compare _container and _ordinal?

This is a reference equality, no need compare any fields. 

---
In reply to: [532883153](https://github.com/dotnet/roslyn/pull/49569#discussion_r532883153) [](ancestors = 532883153)